packages/runtime-common/create-response.ts

```diff
@@ -7,8 +7,9 @@ export function createResponse(
     ...init,
     headers: {
       ...init?.headers,
-      'X-Boxel-Realm-URL': unresolvedRealmURL,
+      'X-Boxel-Realm-Url': unresolvedRealmURL,
       vary: 'Accept',
+      'Access-Control-Expose-Headers': 'X-Boxel-Realm-Url',
```

Comment:
This is needed so I could read the header (`headers.get('x-boxel-realm-url');`). Otherwise it wasn't there